ktoml-core/src/commonTest/kotlin/com/akuleshov7/ktoml/parsers/ValueParserTest.kt

```diff
@@ -18,6 +18,17 @@ class ValueParserTest {
         testTomlValue(Pair("a", "\'false\'"), NodeType.LITERAL_STRING)
     }
 
+    @Test
+    fun dateTimeParsingTest() {
```

Comment:
Seriously, you are the best!

Reply:
I have a little time right now (probably won't be the case for a few weeks after) so I thought that would be a good idea to do simple issues :)